.travis.yml

```diff
@@ -29,6 +29,7 @@ before_script:
 - gfortran --version
 
 script:
+- git config submodule.externals/kokkos.url https://github.com/kokkos/kokkos.git
```

Comment:
Interesting. Was ssh not working?

Reply:
> Was ssh not working?

Unfortunately, it does not. Ssh access to github requires ssh keys to be set up, even for public repos.

Reply:
Uh, true.